src/langsrv/refsvisitors.go

```diff
@@ -315,8 +315,8 @@ func (d *blockMethodCallVisitor) BeforeEnterNode(n ir.Node) {
 
 	exprType := solver.ExprType(d.ctx.Scope(), d.ctx.ClassParseState(), call.Variable)
 
-	exprType.Iterate(func(typ string) {
-		m, ok := solver.FindMethod(typ, methodName)
+	exprType.Iterate(func(typ meta.Type) {
+		m, ok := solver.FindMethod(typ.String(), methodName)
```

Comment:
How about making API that accepts types actually accept types and not require this .String() thing? Basically almost every time you call .String() you want the function to accept the meta.Type instead.

Reply:
I'll take a closer look. 
However, for `FindMethod`, this does not seem like a good idea, since the method takes the name of the class, not its type. Elsewhere we will have to cast the `string` to `Type`.